docs/conf.py

```diff
@@ -30,7 +30,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 #extensions = ['sphinx.ext.autodoc']
 
-extensions = ['djangocms', 'sphinx.ext.intersphinx', 'sphinx.ext.todo']
+extensions = ['djangocms', 'sphinx.ext.intersphinx']
```

Comment:
I'm backporting the fix in 3.2
I think that such test-related fixes should be first applied to earlier versions and then  forwardported, or at least they should be advertised as needing for backporting, otherwise they get lost